R/default-scoring-rules.R

```diff
@@ -0,0 +1,155 @@
+#' @title Select Scoring Rules From A List of Possibilities
+#' @description Helper function to return only the scoring rules selected by
+#' the user from a list of possible scoring rules.
+#' @param possibilities A list of scoring rules.
+#' @param select A character vector of scoring rules to select from the list.
+#' If `select = all` (the default), all possible scoring rules are returned.
+#' @param exclude A character vector of scoring rules to exclude from the list.
+#' If `select != "all"`, this argument is ignored.
+#' @return A list of scoring rules.
+#' @keywords internal
+#' @importFrom checkmate assert_subset assert_list
+#' @examples
+#' scoringutils:::select_rules(
+#'   possibilities = rules_binary(),
+#'   select = "brier_score"
+#' )
+select_rules <- function(possibilities, select, exclude = NULL) {
```

Comment:
having one argument be select and one being exclude seems quite clunky naming-wise. Why not include and exclude?

Reply:
makes sense, I didn't think of that. I'll update the names

Reply:
hm. thinking of it, "include" is not exclusive - while select is. I.e. if you say "I select option A" it means "you pick only A". But if you include A, then others may be present as well. What do you think?

Reply:
yeah okay I think you are right and that the current naming makes sense.